src/options/bv_bitblast_mode.h

```diff
@@ -68,12 +68,33 @@ enum SatSolverMode
   SAT_SOLVER_CADICAL,
 }; /* enum SatSolver */
 
+/**
+ * When the BV solver does eager bitblasting backed by Cryptominisat, proofs
```

Comment:
Nit: CryptoMiniSat (we try to be consistent with the tool names :-))

Reply:
:+1: